torch/distributed/_composable/_ddp.py

```diff
@@ -1166,6 +1167,15 @@ def forward(self, *inputs, **kwargs):
             )
         return output
 
+    def forward(self, *inputs, **kwargs):
```

Comment:
if this `_ddp.py` file is just for composable API, do we even need this forward method?

Reply:
Will clean it up soon